### ISSUE TITLE
Simpler Submissions

### ISSUE THREAD

**Issue by p5quared** [juxtopposed/stacksorted#19]
It might be worth considering simplifying the submission method from gh PR -> form on web-page.

**Comment by juxtopposed** [comment]
Hey there. Do you mean something like this (the one we're using on the site atm)? https://forms.gle/9ruKMLksn98LVd4K9

**Comment by aidrecabrera** [comment]
@juxtopposed. I think @p5quared meant adding actual forms, instead of Google Forms. This would be a great feature in the future, as it would make it easier for users to submit their proposals.

Google Forms is great, but it can be a bit clunky to use (I believe). Having forms directly would be more user-friendly and would make the submission process more streamlined. What do you think?

**Comment by p5quared** [comment]
Yes that is right, I meant to say a native (`<form />`) solution.
Users could submit as much of the requested information as they please.

Setting up [Pocketbase](https://pocketbase.io) to store pending submissions could be straightforward, and more extendable than GForms.

**Comment by juxtopposed** [comment]
I can definitely see this happening at some point. Would love to look into it.
